configure.in

```diff
@@ -698,6 +698,11 @@ WX_ARG_FEATURE(sockets,       [  --enable-sockets        use socket/network clas
 WX_ARG_FEATURE(ipv6,          [  --enable-ipv6           enable IPv6 support in wxSocket], wxUSE_IPV6)
 WX_ARG_FEATURE(ole,           [  --enable-ole            use OLE classes (Win32 only)], wxUSE_OLE)
 WX_ARG_FEATURE(dataobj,       [  --enable-dataobj        use data object classes], wxUSE_DATAOBJ)
+WX_ARG_FEATURE(webrequest,    [  --enable-webrequest     use wxWebRequest], wxUSE_WEBREQUEST)
+WX_ARG_FEATURE(webrequestcurl, [  --enable-webrequest-curl use libcurl with wxWebRequest], wxUSE_WEBREQUEST_LIBCURL)
```

Comment:
Yes that's the idea. It might make sense for some scenarios to use curl on Windows or macOS e.g. to support TLS 1.1 on WinXP or for macOS 10.7 and 10.8 that don't support NSURLSession